PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+- [Description](#description)
+  - [Type of change](#type-of-change)
+- [How Has This Been Tested?](#how-has-this-been-tested)
+- [Checklist](#checklist)
+
+
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.
+List an dependencies that are required for this change.
+
+Fixes # (issue number)
+
+## Type of change
+
+Please delete options that are not relevant
+- [] Bug fix (non-breaking change which fixes an isssue)
+- [] New Feature (non-breaking change which adds functionlity)
+- [] Breaking change(fix or feature that would cause existing functionlity to not work as expected)
+- [] This change requires a documentation update
+
+# How Has This Been Tested?
```

Comment:
I'm not sure how I would fill out this bit - it would be good to understand with an example if we are keeping this part of the template.  It might be worth have an example of a filled out template as well as a blank template to help people that are new to project and new to collaborative working.